### PR TITLE
Features/Metrics project

### PR DESCRIPTION
**Pull request by CesarHerr**
Interactions

Home page

- [x] When the page loads, the webapp shows the list of categories that could be filtered out by some parameter, for example by the category name.
- [x] Along with the category name, you will display some numeric values per category that come from the API.
- [x] When the user clicks (or taps) on a category item, the application navigates to the details page.

Details page
- [x] In the details page, the webapp retrieves data from the API to show detailed data in the category.
- [x] When the user clicks on the "Back" button (<), the user navigates to the home page.

Testing requirements 
- [x] Create unit tests for pure functions in your Redux code.
- [x] Create integration tests for your application using the React Testing Library.
- [x] You may need to mock the access to the API, so that your tests don't send actual requests.
- [x] You may need to mock the connection to the Redux Store.

Technical requirements
- [x] The project is a single-page application (SPA) built with React and Redux.
- [x] The data retrieved from the API should be stored in the Redux store.
- [x] You should filter the data that you retrieve from the API using a Filter stateless component.
- [x] Every page (the main page and the pages for each item) should have a unique route within the SPA.
- [x] The project should be deployed and accessible online.
- [x] Add a link to the online version of your app to your README file.
- [x] Project documentation

Once you have finished the development of the project, you should record a video presenting the features of the project you built. It is a video with a maximum length of 5 minutes. The content of the video should include:
 
- [x] a description of the project.
- [x] a demo of the project features.
- [x] You should also highlight some interesting piece of code or something you built that you are very proud of.

[BotwCapstone.webm](https://github.com/CesarHerr/metrics_webapp_botw/assets/111262493/49b93d74-c9ee-4b93-87e0-968f7c2c2853)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,190 @@
...
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, give it a ⭐️!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+We would like to thank Microverse for giving us the opportunity to learn and grow as developers and also we like to thank our families, they are all our support. 🌟
+
```
Comment (usorfaitheloho) [inline README.md]: In the acknowledgment section, kindly note that you are to acknowledge the author of the original design as recommended by Creative Common License 👍 , The original author is [Nelson Sakwa](https://www.behance.net/sakwadesignstudio)

File: README.md [hunk 2]
```diff
@@ -1,70 +1,190 @@
...
 
-## Learn More
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+[BotwCapstone.webm](https://github.com/CesarHerr/metrics_webapp_botw/assets/111262493/cccc26b3-5b03-46f2-b453-a20b7d67bf91)
 
-To learn React, check out the [React documentation](https://reactjs.org/).
 
-### Code Splitting
+## 🚀 Live Demo <a name="live-demo"></a>
```
Comment (usorfaitheloho) [inline README.md:79]: Your project takes a really long time load on the live link and gives an error when you try to load from liveserver, please can you fix this 😄 

image

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
> Hi @CesarHerr  ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

  [ ](https://camo.githubusercontent.com/679757134a8fee76cfe1941f9f3ecb0e0cf56dd02da4498268ce31f0dacdfca3/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)
> 
> ## Highlights
> * Project contains the home and details page ✔️
> * Has different unit test using jest ✔️
> * Good project documentation ✔️
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (CesarHerr) [reply README.md]: @usorfaitheloho  Thanks!! :bow:  I forget it :facepalm: , I'll do it right now!

File: README.md (hunk 2 above)
Comment (CesarHerr) [reply README.md:79]: @usorfaitheloho  thanks for your review!, I'll check it :+1:

File: README.md (hunk 1 above)
Comment (Whoistolu) [reply README.md]: - Well resolved 👍🏽

File: README.md (hunk 2 above)
Comment (Whoistolu) [reply README.md:79]: - Well resolved 👍🏽

**Review by Whoistolu — APPROVED** [review]
### Hi @CesarHerr 

Great job fixing the recommended changes by the previous reviewer 👍🏽 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by CesarHerr** [comment]
@Whoistolu  Thanks for your review!!! :smile:

**Pull request merged**